plugins/server/org.jetbrains/postgres/2.2/manifest.ktor.yaml

```diff
@@ -9,4 +9,5 @@ prerequisites:
 installation:
   default: install.kt
   outside_app: outside_app.kt
-  source_file_kt: CitySchema.kt
+sources:
+  - file: CitySchema.kt
```

Comment:
We also can support mustache templates in the future to make generated files customizable

Reply:
Oh yeah, good idea!